websites/K/KEXP/presence.ts

```diff
@@ -0,0 +1,67 @@
+const presence = new Presence({ clientId: "1138970637440917504" }),
+	supportedPaths = [
+		"/archive/",
+		"/podcasts/",
+		"/shows/",
+		"/djs/",
+		"/schedule/",
+	],
+	timeElapsed = Math.floor(Date.now() / 1000);
+async function updatePresence() {
+	const strings = await presence.getStrings({
+			pause: "general.paused",
+			play: "general.playing",
+			profile: "general.readingAbout",
+			reading: "general.readingAnArticle",
+			watching: "general.watching",
+		}),
+		currentPagePath = window.location.pathname,
```

Comment:
This repository prefers `document.location` over `window.location`
```suggestion
		currentPagePath = document.location.pathname,
```